src/lib/dates.ts

```diff
@@ -42,8 +42,7 @@ export const DEFAULT_DURATIONS: DurationOption[] = [
   { label: 'Days ago', value: 'day' },
 ];
 
-export const CUSTOM_DATE_RANGE_LABEL_SEPARATOR =
-  '<i aria-hidden="true" class="fa fa-arrow-right"></i>';
+export const CUSTOM_DATE_RANGE_LABEL_SEPARATOR = '<span>&#8594;</span>'; //use utf8 arrow symbol
```

Comment:
A simpler refactoring suggestion:
we could just use space, arrow, space (`&nbsp;&#8594;&nbsp;`)
and remove the styling around the arrow, and drop the v-html

It would differ minimally from the mock-up (the color the arrow would be the same than the dates) though. Do you think it's worth the simplification?

Reply:
Let's try it :wink:

Reply:
Tried it but we cant' because the arrow is 'rikiki' so we need some style :cry: